Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml

```diff
@@ -273,6 +272,15 @@
             <scope>runtime</scope>
         </dependency>
 
+        <!-- xalan dependency required for XSL transformation -->
+        <dependency>
+            <groupId>xalan</groupId>
+            <artifactId>xalan</artifactId>
+            <version>2.7.1</version>
```

Comment:
yes I do have issues with xalan 2.7.2 version which is defined in dependency management for CONNECT.